rust/src/smb/smb1.rs

```diff
@@ -1023,7 +1031,7 @@ pub fn smb1_write_request_record<'b>(state: &mut SMBState, r: &SmbRecord<'b>, an
     smb1_request_record_generic(state, r, events);
 }
 
-pub fn smb1_read_response_record<'b>(state: &mut SMBState, r: &SmbRecord<'b>, andx_offset: usize)
+pub fn smb1_read_response_record<'b>(state: &mut SMBState, r: &SmbRecord<'b>, andx_offset: usize, more_nbss: usize)
```

Comment:
I find the `more_nbss` variable name not clear.

Reply:
Proposing `nbss_remaining`
Do you have a better name ?